packages/core/src/components/account/orders/MyAccountListOrders/MyAccountFilterSlider/section.module.scss

```
@@ -1,23 +1,25 @@
-  @import "@faststore/ui/src/components/molecules/Toggle/styles.scss";
-  @import "@faststore/ui/src/components/molecules/DiscountBadge/styles.scss";
-  @import "@faststore/ui/src/components/molecules/Accordion/styles.scss";
-  @import "@faststore/ui/src/components/molecules/InputField/styles.scss";
-  @import "@faststore/ui/src/components/molecules/Modal/styles.scss";
-  @import "@faststore/ui/src/components/molecules/SelectField/styles.scss";
-  @import "@faststore/ui/src/components/organisms/Filter/styles.scss";
-  @import "@faststore/ui/src/components/organisms/FilterSlider/styles.scss";
-  @import "@faststore/ui/src/components/organisms/SlideOver/styles.scss";
-  @import "./MyAccountFilterFacetDateRange/styles.scss";
-  @import "./MyAccountFilterFacetPendingApproval/styles.scss";
+  @include meta.load-css("~@faststore/ui/src/components/atoms/Button/styles.scss");
```

Stylelint (17.2.0)

[error] Unexpected extension ".scss" in `@meta.load-css` (scss/load-partial-extension)

(scss/load-partial-extension)